attestation-container/uvm/uvm_endorsement.go

```diff
@@ -0,0 +1,22 @@
+package uvm
+
+import (
+	"fmt"
+	"os"
+)
+
+const DEFAULT_UVM_ENDORSEMENT_ENV_VAR_NAME = "UVM_REFERENCE_INFO"
+
+/*
+Fetch UVM endorsement from environment variable as base64 encoded string.
+*/
+func FetchUVMEndorsement(envVarName string) (string, error) {
```

Comment:
"Fetch" feels like the attestation container is fetching this from a remote server, but it's simply looking up an envvar. Should this be `ParseUVMEndorsement` instead, for consistency with the report endorsement?